gcc/rust/checks/errors/rust-feature.cc

```diff
@@ -54,6 +57,7 @@ const std::map<std::string, Feature::Name> Feature::name_hash_map = {
   {"decl_macro", Feature::Name::DECL_MACRO},
   // TODO: Rename to "auto_traits" when supporting
   // later Rust versions
+  {"negative_impls", Feature::Name::NEGATIVE_IMPLS},
```

Comment:
```suggestion
  {"negative_impls", Feature::Name::NEGATIVE_IMPLS},
  // TODO: Rename to "auto_traits" when supporting
  // later Rust versions
```

so that the comment still concerns the right feature gate